cpp/tests/utilities/column_utilities.hpp

```diff
@@ -187,3 +187,34 @@ inline std::pair<thrust::host_vector<std::string>, std::vector<bitmask_type>> to
 
 }  // namespace test
 }  // namespace cudf
+
+// Macros for showing line of failure.
+#define CUDF_TEST_EXPECT_COLUMN_PROPERTIES_EQUAL(lhs, rhs) \
+  {                                                        \
+    SCOPED_TRACE(" <--  line of failure\n");               \
```

Comment:
What is this?

Reply:
when `cudf::test::expect_column_properties_equal` fails, it doesn't show line of error in the test code. it shows column_utilities.cu. This will point to the line of failure. I presume, other devs also find this annoying while writing tests.

Reply:
I will initiate another PR to replace all usage of these APIs with these macros. It helps find test failures lines.

Reply:
Where is SCOPED_TRACE defined?